src/groovy/au/org/emii/gogoduck/job/JobPresenter.groovy

```diff
@@ -27,6 +29,10 @@ class JobPresenter extends HashMap {
         put('queuePosition', jobExecutorService.getQueuePosition(job) + 1)
     }
 
+    void addStatusUrl(job, createLink) {
```

Comment:
I think this could be renamed to `addUrl`, because it's the url to the job, not just its status.